src/dbus/common/constants.hpp

```diff
@@ -46,6 +46,7 @@
 
 #define OTBR_DBUS_SCAN_METHOD "Scan"
 #define OTBR_DBUS_ATTACH_METHOD "Attach"
+#define OTBR_DBUS_DETACH_METHOD "Detach"
```

Comment:
update https://github.com/openthread/ot-br-posix/blob/main/src/dbus/server/introspect.xml?

Reply:
It's already updated <https://github.com/openthread/ot-br-posix/pull/767/files#diff-64258c71b54a165182be9a632cb0e131147fa82efdce34730e5a5c1b40c58c3bR49-R52>